guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<p>Data in a Spatially enabled DataFrame can be exported to JSON based formats, such as <a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=query#arcgis.features.FeatureSet" target="_blank">FeatureSet</a> or <a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html#featurecollection" target="_blank">FeatureCollection</a>, using the <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=to_featurelayer#arcgis.features.GeoAccessor.to_featureset" target="_blank">to_featureset()</a></code> and <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.features.toc.html?highlight=to_featurelayer#arcgis.features.GeoAccessor.to_feature_collection" target="_blank">to_feature_collection()</a></code> methods. Let's take a look.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>